app/src/main/java/com/android/voyageur/ui/profile/ProfileScreen.kt

```diff
@@ -151,8 +159,88 @@ fun ProfileContent(
             onSignOut = onSignOut,
             onEdit = onEdit)
         Spacer(modifier = Modifier.height(20.dp))
-        // Expandable Friend Requests Menu
+
+        // Friend Requests Menu
         FriendReqMenu(friendRequests, notificationUsers, userViewModel)
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        // Trip Invite Menu
+        TripInviteMenu(tripInvites = tripInvites, tripsViewModel = tripsViewModel)
+      }
+}
+
+@Composable
+fun TripInviteItem(tripInvite: TripInvite, tripsViewModel: TripsViewModel) {
+  Row(
+      modifier = Modifier.fillMaxWidth().padding(8.dp).testTag("tripInvite"),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.SpaceBetween) {
+        Text(text = "From: ${tripInvite.from}", modifier = Modifier.weight(1f))
```

Comment:
Here could you replace the hardcoded string please?

Reply:
Done, thanks for pointing this out!